pages/national_resources/uk_resources.md

```diff
@@ -77,7 +77,7 @@ This page provides an overview of the data management resources in the UK. The t
   * UK [Data Protection Act](https://www.gov.uk/data-protection) (2018)
```

Comment:
Thanks @robertmand It would may be helpful to write a short description on what the act is about (and if it differs in any way from GDPR). Also, a short description about TREs doc would be helpful i.e. what sort of principles and best practices are addressed. Cheers